rero_ils/modules/patrons/views.py

```diff
@@ -102,28 +104,45 @@ def profile(viewcode):
     patron = Patron.get_patron_by_user(current_user)
     if patron is None:
         raise NotFound()
+    if request.method == 'POST':
+        loan = Loan.get_record_by_pid(request.values.get('loan_pid'))
+        item = Item.get_record_by_pid(loan.get('item_pid'))
+        data = {
+            'item_pid': item.pid,
+            'pid': request.values.get('loan_pid'),
+            'transaction_location_pid': item.location_pid
+        }
+        try:
+            item.extend_loan(**data)
+            flash(_('The item %(item_id)s has been renewed.',
```

Comment:
as you have new translation, please extract the corresponding translations.

Reply:
closing this comment, as discussed